www/src/pages/tutorial.js

```diff
@@ -4,47 +4,23 @@ import Link from "gatsby-link"
 
 import Container from "../components/container"
 
-export default () => (
+export default (props) => (
```

Comment:
The changes here should be reverted

Reply:
Or actually, this file should be removed since docs/tutorial/index.md is what is being used https://deploy-preview-4153--gatsbyjs.netlify.com/tutorial/